bin/install-package-tests

```diff
@@ -15,11 +15,18 @@ is_numeric() {
         *) return 0;;            # returns 0 if numeric
     esac
 }
-# Promt color vars.
+# Prompt color vars.
 C_RED="\033[31m"
 C_BLUE="\033[34m"
 NO_FORMAT="\033[0m"
 
+# If running in CI, don't use colors.
```

Comment:
GitHub Actions escapes ANSI color codes as expected, so not sure if this is necessary. Additionally, color can help reduce cognitive load when identifying failures or warnings.

Reply:
Indeed, over at https://github.com/wp-cli/scaffold-command/pull/356 it seems to work too.